File: no-auth/src/main/java/com/kjj/noauth/dto/jwt/TokenRefreshResponseDto.java
package com.kjj.noauth.dto.jwt;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class TokenRefreshResponseDto {
    private String message;

    public static TokenRefreshResponseDto refreshSuccess() {
        return new TokenRefreshResponseDto(
                "재발급에 성공했습니다."
        );
    }
    public static TokenRefreshResponseDto refreshFail(Exception e) {
        return new TokenRefreshResponseDto(
                e.getMessage()
        );
    }
}

File: no-auth/src/main/kotlin/com/kjj/noauth/dto/jwt/TokenRefreshResponseDto.kt
package com.kjj.noauth.dto.jwt

data class TokenRefreshResponseDto(
    val message: String
) {
    companion object {
        fun refreshSuccess(): TokenRefreshResponseDto {
            return TokenRefreshResponseDto("재발급에 성공했습니다.")
        }

        fun refreshFail(e: Exception): TokenRefreshResponseDto {
            return TokenRefreshResponseDto(e.message!!)
        }
    }
}